ironic_tempest_plugin/tests/scenario/ironic_standalone/test_basic_ops.py

```diff
@@ -162,6 +162,48 @@ def test_ip_access_to_server(self):
         self.boot_and_verify_node()
 
 
+class BaremetalIdracDriverDirectWholedisk(bsm.BaremetalStandaloneScenarioTest):
+
+    api_microversion = '1.31'  # to set the deploy_interface
+    driver = 'idrac'
+    deploy_interface = 'direct'
+    image_ref = CONF.baremetal.whole_disk_image_ref
+    wholedisk_image = True
+
+    @utils.services('image', 'network')
+    @decorators.idempotent_id('c2bebda2-fd27-4b10-9015-f7d877f0eb60')
+    def test_ip_access_to_server(self):
+        self.boot_and_verify_node()
+
+
+class BaremetalRedfishDriverDirectWholedisk(bsm.BaremetalStandaloneScenarioTest):
```

Comment:
Duplicate Test case. Please check for :
https://github.com/openstack/ironic-tempest-plugin/blob/3d2de82b3e08b348732156ab95a492cd40022e17/ironic_tempest_plugin/tests/scenario/ironic_standalone/test_basic_ops.py#L424